Makefile.am

```diff
@@ -396,7 +398,8 @@ test_apps = \
 	tests/test_sockopt_hwm \
 	tests/test_heartbeats \
 	tests/test_stream_exceeds_buffer \
-	tests/test_pub_invert_matching
+	tests/test_pub_invert_matching \
+	tests/test_dgram
```

Comment:
Please add this in the "if ENABLE_DRAFTS" section below, otherwise it will break the build